### PR TITLE
[SettingsControls] Page sample tweak

### PR DESCRIPTION
**Pull request by niels9001**
Tweaked the Settings Page sample so it can be referenced by developers that want to create a consistent settings page.

image

### CODE REVIEW

File: labs/SettingsControls/samples/SettingsControls.Samples/SettingsPageExample.xaml [hunk 1]
```diff
@@ -5,155 +5,109 @@
...
+                       Text="About" />
 
-            <labs:SettingsExpander Description="The SettingsExpander has the same properties as a Card, and you can set SettingsCard as part of the Items collection."
-                                   Header="SettingsExpander">
+            <labs:SettingsExpander Description="© 2023. All rights reserved."
+                                   Header="Community Toolkit Gallery">
                 <labs:SettingsExpander.HeaderIcon>
-                    <FontIcon Glyph="&#xE91B;" />
+                    <BitmapIcon ShowAsMonochrome="False"
+                                UriSource="https://raw.githubusercontent.com/CommunityToolkit/Labs-Windows/main/common/CommunityToolkit.Labs.Shared/Assets/AppTitleBar.scale-200.png" />
```
[the remark this answers is not in the document]
Comment (niels9001) [reply labs/SettingsControls/samples/SettingsControls.Samples/SettingsPageExample.xaml]: Oooh didn't know that'd work.. fixed :)

**Comment by michael-hawker** [comment]
Unrelated to this PR, we'll have to figure out what in the Expander is causing Uno WASM to break here:

image

Though the example page isn't loading at all now in WASM (compared to it loading fine on toolkitlabs.dev [outside of the expander issue]):

image

Could it be the Entrance Theme Transition? Maybe we need to guard that with a `<win:`?

FYI @Arlodotexe @jeromelaban

**Comment by niels9001** [comment]
@michael-hawker Ah, forgot to push the guards commit 🤦‍♂️. 

Yeah, the Expander not loading its SettingsCard is weird.. I'll check tomorrow to see if I can remove enough to see if the Expander header content wants to load the Card at all.

**Comment by michael-hawker** [comment]
Thanks @niels9001, loading now on WASM with the guard. Let me know if you find which bit is triggering WASM and we can open an Uno bug (or find if there's an existing issue to track).

It's looking great on UWP! I didn't know about the `EntranceThemeTransition` trick, that's a new one to me. 🙂

The animation looks wonky on WinUI 3 though a bit (they blink more than just nicely fly in); we'll have to see if when we update to WinAppSDK 1.2 if it gets better or open an issue on the platform.

**Review by michael-hawker — APPROVED** [review]

**Pull request merged**